examples/textured-floor/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>A-Frame Lounge Component - Floor</title>
+    <meta name="description" content="Basic example for Lounge Floor component."></meta>
+    <script src="https://aframe.io/releases/1.0.4/aframe.min.js"></script>
+    <script src="../../index.js"></script>
+<!--    <script src="../../dist/aframe-lounge-component.min.js"></script> -->
+  </head>
+  <body>
+    <a-scene>
+      <a-entity lounge-floor="texture: textures/exampleTexture.jpg"></a-entity>
```

Comment:
Please, move `textures/exampleTexture.jpg` to `examples/foor/floor-texture.jpg`, to have everything together. Maybe we can move it to some common dir later, if we use it for some other examples.